.github/workflows/check.yml

```diff
@@ -0,0 +1,74 @@
+# This workflow will build the project with Gradle, run integration tests, and release.
+# Because secrets are not available on external forks, this job is expected to fail
+# on external pull requests.
+
+name: Build, Check, Publish
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Gradle Wrapper Validation
+      uses: gradle/wrapper-validation-action@v1
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'zulu'
+
+    - name: Grant execute permissions
+      run: chmod +x gradlew
+        && chmod +x bin/maven
+        && chmod +x run-build
+        && chmod +x scripts/check-clean-git-status
+
+    - name: Run Tests and Build
+      run: ./run-build
+
+    - name: Ensure no changes in Generated Code
+      run: ./scripts/check-clean-git-status
+
+  publish:
+    runs-on: ubuntu-latest
+    if: github.repository == 'hellosign/dropbox-sign-java' && github.ref == 'refs/heads/main' && github.event_name != 'pull_request'
+    needs: [build]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Install JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Upload Artifacts
+        run: ./gradlew publishAllPublicationsToMavenCentralRepository --no-daemon --no-parallel --no-configuration-cache --stacktrace
+        env:
+          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.SONATYPE_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.SONATYPE_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.SIGNING_KEY }}
+          ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.SIGNING_PASSWORD }}
+
+      - name: Retrieve version
+        run: |
+          echo "VERSION_NAME=$(cat gradle.properties | grep -w "VERSION_NAME" | cut -d'=' -f2)" >> $GITHUB_ENV
```

Comment:
the `VERSION_NAME` is not in `gradle.properties` yet (currently it's in `build.gradle`. Let me create a PR to fix it

Reply:
https://github.com/hellosign/hellosign-openapi/pull/235